clients/tabby-agent/src/TabbyAgent.ts

```diff
@@ -144,7 +144,7 @@ export class TabbyAgent extends EventEmitter implements Agent {
     if (options.config) {
       await this.updateConfig(options.config);
     }
-    await this.anonymousUsageLogger.event("Initialize Agent", {
+    await this.anonymousUsageLogger.event("Agent:Initialize", {
```

Comment:
Pascal case means `AgentInitialized`, there shoud not be `:`

Reply:
Fixed